src/lib/signature_lib/test/signature_lib_tests.ml

```diff
@@ -1,43 +1,67 @@
+let signature_expected = 
+  ( Snark_params.Tick.Field.of_string
+      "22392589120931543014785073787084416773963960016576902579504636013984435243005"
+  , Snark_params.Tock.Field.of_string
+      "21219227048859428590456415944357352158328885122224477074004768710152393114331"
+   )
 
-    let signature_expected =
-      ( Snark_params.Tick.Field.of_string
-          "22392589120931543014785073787084416773963960016576902579504636013984435243005"
-      , Snark_params.Tock.Field.of_string
-          "21219227048859428590456415944357352158328885122224477074004768710152393114331"
-      )
+let test_signature_of_empty_random_oracle_input_matches () =
+  let signature_got =
+    Schnorr.Legacy.sign ~signature_kind privkey
+      (Random_oracle_input.Legacy.field_elements [||])
+  in
+  Alcotest.(check bool) "signature of empty random oracle input matches" true
+    (Snark_params.Tick.Field.equal (fst signature_expected) (fst signature_got)
+    && Snark_params.Tock.Field.equal (snd signature_expected) (snd signature_got))
 
-    let%test "signature of empty random oracle input matches" =
-      let signature_got =
-        Schnorr.Legacy.sign ~signature_kind privkey
-          (Random_oracle_input.Legacy.field_elements [||])
-      in
-      Snark_params.Tick.Field.equal (fst signature_expected) (fst signature_got)
-      && Snark_params.Tock.Field.equal (snd signature_expected)
-           (snd signature_got)
+let test_signature_of_signature_matches () =
+  let signature_got =
+    Schnorr.Legacy.sign ~signature_kind privkey
+      (Random_oracle_input.Legacy.field_elements
+         [| fst signature_expected |] )
+  in
+  let signature_expected =
+    ( Snark_params.Tick.Field.of_string
+        "7379148532947400206038414977119655575287747480082205647969258483647762101030"
+    , Snark_params.Tock.Field.of_string
+        "26901815964642131149392134713980873704065643302140817442239336405283236628658"
+    )
+  in
+  Alcotest.(check bool) "signature of signature matches" true
+    (Snark_params.Tick.Field.equal (fst signature_expected) (fst signature_got)
+    && Snark_params.Tock.Field.equal (snd signature_expected) (snd signature_got))
 
-    let%test "signature of signature matches" =
-      let signature_got =
-        Schnorr.Legacy.sign ~signature_kind privkey
-          (Random_oracle_input.Legacy.field_elements
-             [| fst signature_expected |] )
-      in
-      let signature_expected =
-        ( Snark_params.Tick.Field.of_string
-            "7379148532947400206038414977119655575287747480082205647969258483647762101030"
-        , Snark_params.Tock.Field.of_string
-            "26901815964642131149392134713980873704065643302140817442239336405283236628658"
-        )
-      in
-      Snark_params.Tick.Field.equal (fst signature_expected) (fst signature_got)
-      && Snark_params.Tock.Field.equal (snd signature_expected)
-           (snd signature_got)
-  end )
+let test_signature_of_fields_123_matches () =
```

Comment:
It seems to be a regression test. Can you add the commit it has been used to generate the value, if it comes from this repository?

Something like
```
The expected value has been generated using the commit [...] of [this repository/repository X Y Z]
```